subjects/addifpositive/README.md

```diff
@@ -0,0 +1,39 @@
+## add-if-positive 
+
+### Instructions
+Write a function that takes two numbers and adds them together if they are both positive.
+- If either number is negative or one of them is negative and the other is positive return `0`.
+
+### Expected function 
+```go
+func AddIfPositive(a int, b int) int {
+    // your code here
+}
+```
+
+### Usage
+
+```go
+package main
+
+import "fmt"
+
+func main() {
+    fmt.Println(AddIfPositive(1, 2))
+    fmt.Println(AddIfPositive(1, -2))
+    fmt.Println(AddIfPositive(-1, 2))
+    fmt.Println(AddIfPositive(-1, -2))
+    fmt.Println(AddIfPositive(10,20))
```

Comment:
I think it is not clear what is supposed to happen when you add the number 0 with another number. Consider adding where in the example.

Reply:
the subject says that if both numbers are positive it should return the sum of both numbers,
so even if one of them equals 0 or both of them are 0.

Reply:
From what I understand, the number Zero is neither positive nor negative. However, Zero is a positive integer since it doesn't carry any negative sign. So it might do a bit of confusion because of the term number and integer.